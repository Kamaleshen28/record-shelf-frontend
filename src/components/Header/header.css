
.header{
    background-color: #38497E;
    color: white;
    height:60px;
    display: flex;
    justify-content: flex-start;
    align-items: center;
    padding-left: 10px;
    font-size: x-large;
}

.record-header{
    font-weight:bolder;
}