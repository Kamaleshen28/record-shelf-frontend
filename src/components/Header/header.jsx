import React from 'react';
import './header.css';

export default function Header(){
    return(
        <div className='header'>
            <p>My <span className='record-header'>Record</span> Shelf</p>
        </div>
    );
}