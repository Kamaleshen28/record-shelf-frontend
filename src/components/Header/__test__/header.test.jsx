import { render, screen, waitFor } from '@testing-library/react';
import React from 'react';
import Header from '../header';

describe('Header', () => {
    it('should render header correctly', async () => {
        const {asFragment} = render(<Header />);
        expect(asFragment()).toMatchSnapshot();
        // await waitFor(() => {
        //     expect(screen.getByText('Shelf')).toBeTruthy();
        // });
    });
});