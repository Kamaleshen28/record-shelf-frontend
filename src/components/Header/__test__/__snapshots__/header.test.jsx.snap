// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render header correctly 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <p>
      My 
      <span
        class="record-header"
      >
        Record
      </span>
       Shelf
    </p>
  </div>
</DocumentFragment>
`;
